packages/verification-vue-component/src/locales/de.json

```diff
@@ -1,78 +1,123 @@
+                    },
+                    {
+                        "title": "Wie wird der Datenschutz gewährleistet?",
+                        "content": "Zur Überprüfung muss man zwingend im Besitz des Dokuments sein, ansonsten kann man keinerlei Informationen aus der Blockchain gewinnen. Das Dokument wird zur Überprüfung niemals hochgeladen, sondern direkt auf dem Gerät des Benutzers analysiert."
+                    },
+                    {
+                        "title": "Wie werden die Identitäten verifiziert?",
+                        "content": "Die Certifaction-Technologie wird von verschiedenen innovativen Anbietern genutzt, die ihre Kunden jeweils genau prüfen, bevor sie ihre Identität verifizieren. Für Details können Sie sich direkt an den Anbieter wenden (siehe \"Identität verifiziert durch\")."
+                    },
+                    {
+                        "title": "Wie hoch ist die Beweiskraft?",
+                        "content": "Diese Lösung bietet eine verifizierte Identität des Herausgebers, geschützte Benutzerkonten und einen kryptografisch überprüfbaren Beweis der Signatur."
+                    }
+                ]
+            }
         },
         "result": {
-            "title": "Verifizierungsresultat",
+            "title": "Prüfresultat",
             "expertInfo": {
-                "hide": "Experten-Info ausblenden",
-                "show": "Experten-Info anzeigen"
+                "hide": "Technische Details verbergen",
+                "show": "Technische Details anzeigen"
             },
             "issuer": "Aussteller",
             "meta": {
                 "contract": "Contract",
-                "issuer": "Aussteller",
-                "issuerVerifiedBy": "Verifiziert durch",
-                "issuerVerifiedByInformation": "Dieser Dienstleister hat den Aussteller einer gewissenhaften Überprüfung unterzogen. Nur berechtigte Personen dürfen die Verifizierung der Identität ihrer Organisation beantragen.",
-                "issuerAddress": "Ausstelleradresse",
-                "fingerprint": "Dokument-Hashwert",
-                "registrationDate": "Ausstellungsdatum",
-                "registrationTransaction": "Ausstellungstransaktion",
-                "revocationTransaction": "Annullierungstransaktion",
-                "revocationDate": "Annullierungsdatum"
+                "issuer": "AUSSTELLER",
+                "issuerVerifiedBy": "VERIFIZIERT DURCH",
+                "issuerAddress": "PUBLIC KEY DES AUSSTELLERS",
+                "fingerprint": "DOKUMENTEN-HASH",
+                "registrationDate": "AUSSTELLDATUM & -ZEIT",
+                "registrationTransaction": "AUSSTELLUNGSTRANSAKTION",
+                "revocationTransaction": "ANNULLIERUNGSTRANSAKTION",
+                "revocationDate": "ANNULLIERUNGSDATUM & -ZEIT",
+                "smartContractAddress": "SMART CONTRACT-ADRESSE"
```

Comment:
Instead of making it uppercase in the translation, I would use the CSS property `text-transform: uppercase;` to make it uppercase. With that, the visual style is part of the CSS as it should be.

Reply:
Fixed it acc.